bin/make_view

```diff
@@ -198,10 +198,22 @@ CREATE VIEW independent_candidate_expenditures AS
   AND "FPPC" IS NOT NULL
   AND "Cand_NamL" IS NOT NULL;
 
+DROP VIEW IF EXISTS clean_summary;
+CREATE VIEW clean_summary AS
+  SELECT * from "Summary" s
+  WHERE
+  NOT EXISTS
+    (select t."Rpt_Date" from "Summary" t where s."Filer_ID" = t."Filer_ID"
+    and t."From_Date" <= s."From_Date" and s."Thru_Date" <= t."Thru_Date"
+    and (t."From_Date" <> s."From_Date" or s."Thru_Date" <> t."Thru_Date")
+    )
+;
+
+## Remove summary data that is covered by a report that includes that time period
```

Comment:
Is this correct?  I thought comments in sql was `--`.

Reply:
It is not correct.  I thought I had run the script but obviously not, I was a bit rushed at the time.  I pushed a fix.